fgdc_config.rb

```diff
@@ -35,7 +35,7 @@
 to_field 'cho_description', extract_fgdc('/*/idinfo/descript/abstract')
 to_field 'cho_description', extract_fgdc('/*/idinfo/descript/purpose')
 to_field 'cho_description', extract_fgdc('/*/idinfo/status/update')
-to_field 'cho_edm_type', normalize_type
+to_field 'cho_edm_type', literal('image')
```

Comment:
Some of the Harvard materials look like they're GIS data, not just images, e.g. https://spotlight.dlme.clir.org/library/catalog/LANDSCANPOP2008 

I think we need to make a call whether we want delete them.